packages/liferay-npm-scripts/src/utils/set-env.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+function log(message) {
```

Comment:
Whats the idea behind abstracting `console.log` here? Just to avoid the `no-console`?

Also, we should probably just remove the `no-console` rule.

Reply:
Yep that's the sole idea. So I have exactly one `console.log` and the smallest suppression.

Reply:
Maybe it would not be better to disable the `no-console` only in that file?

Reply:
I'd probably vote to just remove the `no-console` rule altogether for this project. It also feels like a bit of overkill to abstract here. 

In the end it isn't a huge deal to me though, fine merging with it as is.

Reply:
As a follow-up I made this: https://github.com/liferay/liferay-npm-tools/pull/92

With that there is only one `console.log` in the entire project, so I probably wouldn't touch the rule.